src/device-registry/validators/measurements.validators.js

```
@@ -0,0 +1,1016 @@
+      .bail()
+      .trim()
+      .toLowerCase()
+      .isIn(constants.NETWORKS)
+      .withMessage("the tenant value is not among the expected ones"),
+  ],
+
+  pagination: (defaultLimit = 1000, maxLimit = 2000) => {
+    return (req, res, next) => {
+      let limit = parseInt(req.query.limit, 10);
+      const skip = parseInt(req.query.skip, 10);
+      if (isNaN(limit) || limit < 1) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)